starter-code/src/app/services/access-control-log.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class AccessControlLogService {
+
+  // Array<Object> but we use 'any' to avoid eslint correction
+  logMessages: any = [{
+      person: String,
+      message: String,
+      createdAt: Date
+    }];
+
+  constructor() { }
+
+  addAccessItem(person: String, message: String) {
+    this.logMessages.person = person;
+    this.logMessages.message = message;
+    console.log(this.logMessages);
```

Comment:
What about the date? More important, what about PUSHING the object into the array? this.logMessages is an array, not an object, so you can't acces to this.logMessages.person.